src/app/core/http-services/joined-groups.service.ts

```diff
@@ -46,6 +52,8 @@ export class JoinedGroupsService {
       );
   }
 
-}
-
+  leave(id: string): Observable<LeaveGroupResponse> {
+    return this.http.delete<LeaveGroupResponse>(`${appConfig().apiUrl}/current-user/group-memberships/${id}`);
+  }
 
+}
```

Comment:
This file was specifically for the service returning joined group. So if you cannot just add this service here.
Probably it is easier to just create a "GroupLeaveService".

Reply:
Fixed